interface-definitions/nat64.xml.in

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="nat64" owner="${vyos_conf_scripts_dir}/nat64.py">
+    <properties>
+      <help>IPv6-to-IPv4 Network Address Translation (NAT64) Settings</help>
+      <priority>500</priority>
+    </properties>
+    <children>
+      <node name="source">
+        <properties>
+          <help>IPv6 source to IPv4 destination address translation</help>
+        </properties>
+        <children>
+          <tagNode name="rule">
+            <properties>
+              <help>Source NAT64 rule number</help>
+              <valueHelp>
+                <format>u32:1-999999</format>
+                <description>Number for this rule</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-999999"/>
+              </constraint>
+              <constraintErrorMessage>NAT64 rule number must be between 1 and 999999</constraintErrorMessage>
+            </properties>
+            <children>
+              #include <include/generic-description.xml.i>
+              #include <include/generic-disable-node.xml.i>
+              <leafNode name="mode">
+                <properties>
+                  <help>Operation mode of Jool packet translator</help>
+                  <valueHelp>
+                    <format>netfilter</format>
```

Comment:
Do you mean nftables?

Reply:
Unfortunately not. Jool doesn't support nftables yet https://github.com/NICMx/Jool/issues/273#issuecomment-1328343927

Thinking about it, vyos provides no way to configure the iptables rules required to make the iptables mode actually useful, so maybe I should drop this mode selector all together (for now, until jool supports nftables?)